### DISCUSSION TITLE
Generate descriptions with GTP-3

### DISCUSSION THREAD

**Discussion by Martin-Milbradt** [eikowagenknecht/lootscraper#11] [Feature Requests]
https://openai.com/api/

Example for prompt "Epic Games - Cities Skylines":

Cities: Skylines is a city-building and management game developed by Colossal Order and published by Paradox Interactive. The game was released for Microsoft Windows, OS X, and Linux on March 10, 2015.

In the game, the player builds a city from scratch, dealing with issues such as traffic, fires, pollution and crime. The player can also manage the city's economy, with the ability to create taxes and to subsidy certain areas of the city.

**Comment by eikowagenknecht** [comment]
Does this work for loot or lesser known games as well?
